URLRequest+RSWeb.swift

```diff
@@ -32,7 +32,14 @@ public extension URLRequest {
         case .readerAPIAuthLogin(_, let apiKey):
             let auth = "GoogleLogin auth=\(apiKey)"
             setValue(auth, forHTTPHeaderField: HTTPRequestHeader.authorization)
-		}
+        case .feedlyAccessToken(_, let token):
+            let auth = "OAuth \(token)"
+            setValue(auth, forHTTPHeaderField: "Authorization")
+        case .feedlyRefreshToken:
+            // https://developer.feedly.com/v3/auth/#refreshing-an-access-token
+            assertionFailure("Refresh tokens stored to refresh the Feedly access token. Did you mean to use `feedlyAccessToken` instead?")
+            break
+        }
```

Comment:
I'm not sure storing and retrieving the refresh token is an appropriate use for the `Credentials` and `CredentialsManager` types.

The idea is to store the refresh token so when the feedly API indicates an access token is stale, the client (e.g.: NNW) can retrieve the refresh token and obtain a fresh access token.

Thoughts appreciated.

Reply:
In 86bc8954bfc1a858d5478c76b99099df0b7a6928, added code comments to explain the awkwardness and note it should be refactored when use cases become clearer.